docs/core/tutorials/using-with-xplat-cli.md

```diff
@@ -147,10 +148,25 @@ Let's build off of the previous Fibonacci example by caching some Fibonacci valu
    377
    ```
 
+## Publish your app
+
+Once you're ready to distribute your app, use the [`dotnet publish`](../tools/dotnet-publish.md) command to generate the _publish_ at _bin\\debug\\netcoreapp2.1\\publish\\_ (use `/` for non-Windows systems). You can distribute the contents of the _publish_ folder to other platforms as long as they've already installed the dotnet runtime.
+
+You can run your published app with the `dotnet` command:
+
+```console
+$ dotnet bin\Debug\netcoreapp2.1\publish\Hello.dll
+Hello World!
+```
+
+## Conclusion
+
 And that's it! Now, you can start using the basic concepts learned here to create your own programs.
 
 Note that the commands and steps shown in this tutorial to run your application are used during development time only. Once you're ready to deploy your app, you'll want to take a look at the different [deployment strategies](../deploying/index.md) for .NET Core apps and the [`dotnet publish`](../tools/dotnet-publish.md) command.
```

Comment:
This no longer makes sense given what you've just added but some of the information on this paragraph should be moved to the previous section.